AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/utils.py

```diff
@@ -155,6 +155,29 @@ def fail_if_log_line_found(window, line, print_infos, time_out):
         Report.critical_result(("Unexpected line not found: " + line, "Unexpected line found: " + line), not TestHelper.find_line(window, line, print_infos))
 
     @staticmethod
+    def all_expected_log_lines_found(section_tracer, lines):
+        """
+        function for parsing game mode's console output for expected test lines. duplicate lines and error lines are not
+        handled by this function.
+
+        param section_tracer: python editor tracer object
+        param lines: list of expected lines
+
+
+        returns true if all the expected lines were detected in the parsed output
+        """
+        found_lines = [printInfo.message.strip() for printInfo in section_tracer.prints]
+
+        expected_lines = len(lines)
+        matching_lines = 0
+
+        for line in lines:
+            for found_line in found_lines:
+                if line == found_line:
+                    matching_lines += 1
+
+        return matching_lines >= expected_lines
+    @staticmethod
```

Comment:
white space

Reply:
added space between the function return and the next function definition